.gitignore

```diff
@@ -51,3 +51,5 @@ android/app/libs
 fastlane/report.xml
 fastlane/Preview.html
 fastlane/screenshots
+
+src/js/keys.js
```

Comment:
@minaorangina as we discussed this is only the temporary solution until we figure out how to properly set the global var and how to hide them - #92